docs/en/observability/uptime-set-up-app.asciidoc

```diff
@@ -0,0 +1,168 @@
+[[uptime-set-up-choose-agent]]
+= Use the {uptime-app}
+
+beta[]
+
+You can now create monitors directly in the {uptime-app}.
+To try out this beta functionality, first you will need to enable Monitor Management:
+
+. Go to **{observability}** -> **Uptime**.
+. Click **Monitor Management**.
+. Review the terms and click **Enable** to start using Monitor Management.
+
+From here, you can either <<global-managed-testing-infrastructure,use global managed testing infrastructure>>
+or <<private-locations,create a private location>> to run monitors from your own premises.
+
+NOTE: To use the Elastic Synthetics integration, see the https://www.elastic.co/guide/en/observability/8.3/uptime-set-up.html#uptime-set-up-choose-agent[8.3 documentation]. The Elastic Synthetics integrations is similar to the {uptime-app} approach, but contains fewer configuration options.
+
+[discrete]
+[[global-managed-testing-infrastructure]]
+== Use global managed testing infrastructure
+
+// Is there better language we can use to describe what it is
+// instead of what is _not_? 
+With Elastic's global managed testing infrastructure, you can create and run monitors in multiple
+locations using the {uptime-app} the UI without having to set up {fleet},
+pull a Docker image, and install and run an agent on your machine.
```

Comment:
Is there better language we can use to describe what it is instead of what is _not_?

Reply:
I'd maybe just contract this to "without the hassle of managing your own infrastructure. We take care of software updates and capacity planning for you". Or something similar, I think you're right to focus on the positive.

Reply:
Agreed, I like that suggestion